doc/graph/fusion_patterns/sdpa.md

```diff
@@ -31,11 +31,11 @@ SDPA graph, getting partition from the graph, and optimizing the kernels
 underneath. In general, an SDPA pattern is defined as a directional acyclic
 graph (DAG) using oneDNN Graph API.
 
-### Floating-point SDPA
+### Floating-point SDPA for Inference and Training Forward
```

Comment:
I would suggest to separate inference and training into two sections. Keep the existing inference part unchanged and in the new training section, explicit mention that training forward pattern is the same (if it is) as inference pattern and refer readers to the inference section.

Reply:
updated, thanks for the suggestion!